providers/catalog.go

```diff
@@ -136,4 +137,25 @@ var catalog = CatalogType{ // nolint:gochecknoglobals
 			Write:     false,
 		},
 	},
+
+	//Close configuration
+	Close: {
+		AuthType: Oauth2,
+		BaseURL:  "https://api.close.com",
```

Comment:
Seeing all requests as `https://api.close.com/api`, are there any modules / endpoints that don't use `api`? If not, we can probably include that in the base URL too.